eras/shelley/impl/cddl-files/shelley.cddl

```diff
@@ -21,38 +21,39 @@ $vrf_vkey = bytes .size 32
 addr_keyhash = $hash28
 
 ; address = bytes
-;     reward_account = bytes
+; reward_account = bytes
```

Comment:
This comment should be near the definition of `reward_account`

Reply:
Thanks! I have moved it as part of its definition. :+1: